### PR TITLE
Supporting vendor assessments

### PR DESCRIPTION
**Pull request by jburger**
This small PR includes a grab bag of improvements to the security pages with the intent of making vendor assessments work as close to the following ideal as it can.

"When a customer asks us about our security posture, everything they need to answer that question can be easily found on our website"

- [x] ensures the CAIQ can be found from the index page. It also adds an image to depict the shared security model. 
- [x] added a visual model to help explain the shared model comparison a little better, for customers considering a move from on-prem to cloud.

### CODE REVIEW

File: docs/security/index.md [hunk 1]
```diff
@@ -10,12 +10,16 @@ This section provides information about the responsibility we take to provide a
 
 Every year Octopus undergoes a security review conducted by a third-party company. The latest report can be downloaded: [security-review.pdf](https://octopus.com/documents/security-review-executive-report.pdf)
 
+We often hear from customers wanting to know more about our security posture. We've performed a [self assessment against various industry standard controls](/docs/security/caiq/index.md). Feel free to use this in any vendor assessments you'd like to perform.
+
 ## Responsibility
 
 Octopus Deploy take responsibility to provide a secure and stable platform for managing your deployments.
```
Comment (wizedkyle) [inline docs/security/index.md:17]: This doesn't sound quite right what about something like:

"Octopus Deploy has the responsibility to provide a secure and stable platform for managing your deployments.

File: docs/security/index.md [hunk 2]
```diff
@@ -10,12 +10,16 @@ This section provides information about the responsibility we take to provide a
...
 
 You take responsibility for how that platform is implemented and exposed to your infrastructure and users.
```
Comment (wizedkyle) [inline docs/security/index.md:19]: Same as above it doesn't sound quite right what about something like:

"You have the responsibility for how that platform is implemented and exposed to your infrastructure and users."

**Review by wizedkyle — APPROVED** [review]
I have added a couple of comments about wording, but otherwise the shared responsibility diagram looks excellent.

File: docs/security/index.md (hunk 1 above)
Comment (jburger) [reply docs/security/index.md:17]: Thanks @wizedkyle I'm not planning on changing that particular part of the documentation, but it is still a valid question. I usually defer to our wordsmith @wordlee for such things - what do you think Lee?

Comment (wordlee) [inline docs/security/index.md:17]: Happy to update it. Slight tweak:
```suggestion
Octopus Deploy has the responsibility of providing a secure and stable platform for managing your deployments.
```

File: docs/security/index.md (hunk 2 above)
Comment (wordlee) [inline docs/security/index.md:19]:
```suggestion
You have the responsibility for how that platform is implemented and exposed to your infrastructure and users.
```

File: docs/security/index.md (hunk 1 above)
Comment (wordlee) [reply docs/security/index.md:17]: Mad ea change based on @wizedkyle's suggestion

File: docs/security/index.md [hunk 3]
```diff
@@ -10,11 +10,15 @@ This section provides information about the responsibility we take to provide a
 
 Every year Octopus undergoes a security review conducted by a third-party company. The latest report can be downloaded: [security-review.pdf](https://octopus.com/documents/security-review-executive-report.pdf)
 
+We often hear from customers wanting to know more about our security posture. We've performed a [self assessment against various industry standard controls](/docs/security/caiq/index.md). Feel free to use this in any vendor assessments you'd like to perform.
```
Comment (wordlee) [inline docs/security/index.md:13]:
```suggestion
We often hear from customers who want to know more about our security posture. We've performed a [self assessment against various industry-standard controls](/docs/security/caiq/index.md). Feel free to use this in any vendor assessments you need to perform.
```

**Review by wordlee — APPROVED** [review]
Looks good. I added a couple of minor tweaks to the text.

**Pull request merged**